### ISSUE TITLE
No version of 'puppet-unattended_upgrades' can satisfy all dependencies

### ISSUE THREAD

**Issue by KpuCko** [voxpupuli/puppet-unattended_upgrades#167]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 5.5.20
- Ruby: N/A
- Distribution: Ubuntu 18.04.4 LTS
- Module version: 4.0.0

## How to reproduce (e.g Puppet code you use)

puppet module install puppet-unattended_upgrades

## What are you seeing

```
root@puppet.home.lan:~# puppet module install puppet-unattended_upgrades
Notice: Preparing to install into /etc/puppetlabs/code/environments/production/modules ...
Notice: Downloading from https://forgeapi.puppet.com ...
Error: Could not install module 'puppet-unattended_upgrades' (???)
  No version of 'puppet-unattended_upgrades' can satisfy all dependencies
    Use `puppet module install --ignore-dependencies` to install only this module
root@puppet.home.lan:~#
```

## What behaviour did you expect instead

The module to be installed and available to use.

## Output log

```
root@puppet.home.lan:~# puppet module install puppet-unattended_upgrades
Notice: Preparing to install into /etc/puppetlabs/code/environments/production/modules ...
Notice: Downloading from https://forgeapi.puppet.com ...
Error: Could not install module 'puppet-unattended_upgrades' (???)
  No version of 'puppet-unattended_upgrades' can satisfy all dependencies
    Use `puppet module install --ignore-dependencies` to install only this module
root@puppet.home.lan:~#
```

## Any additional information you'd like to impart

```
root@puppet.home.lan:~# puppet module list
/etc/puppetlabs/code/environments/production/modules
├── duxklr-manageusers (v1.0.2)
├── example42-puppi (v2.2.11)
├── example42-timezone (v2.0.16)
├── gbrown-yumrepos (v0.0.3)
├── herculesteam-augeasproviders_base (v2.1.0)
├── herculesteam-augeasproviders_core (v2.6.0)
├── herculesteam-augeasproviders_mounttab (v2.1.1)
├── herculesteam-augeasproviders_shellvar (v4.0.0)
├── herculesteam-augeasproviders_ssh (v3.3.0)
├── herculesteam-augeasproviders_sysctl (v2.5.0)
├── nanliu-staging (v1.0.3)
├── pdxcat-nrpe (v2.1.1)
├── puppet-puppetboard (v6.1.0)
├── puppet-python (v3.0.1)
├── puppetlabs-apache (v5.4.0)
├── puppetlabs-apt (v7.4.1)
├── puppetlabs-concat (v6.2.0)
├── puppetlabs-firewall (v2.3.0)
├── puppetlabs-inifile (v3.1.0)
├── puppetlabs-motd (v4.1.0)
├── puppetlabs-mount_providers (v2.0.1)
├── puppetlabs-ntp (v8.3.0)
├── puppetlabs-postgresql (v6.4.0)
├── puppetlabs-puppetdb (v7.4.0)
├── puppetlabs-registry (v3.1.0)
├── puppetlabs-stdlib (v6.3.0)
├── puppetlabs-translate (v2.2.0)
├── puppetlabs-vcsrepo (v3.1.0)
├── razorsedge-snmp (v3.2.0)
├── saz-locales (v2.5.1)
├── saz-sudo (v6.0.0)
├── stahnma-epel (v1.3.1)
└── thias-sysctl (v1.0.6)
/etc/puppetlabs/code/modules (no modules installed)
/opt/puppetlabs/puppet/modules (no modules installed)
root@puppet.home.lan:~#
```

**Comment by ekohl** [comment]
We've just released 5.0.0. Would you mind trying that?

**Comment by KpuCko** [comment]
I tried installing the module today. All went fine.
The module is working as I expect.
Thanks a lot for the efforts.
